test/api_stability/CMakeLists.txt

```diff
@@ -31,16 +31,18 @@ macro (seqan3_api_stability_test test_suite)
         COMMAND ${CMAKE_COMMAND} -E remove_directory "<SOURCE_DIR>/include"
         COMMAND ${CMAKE_COMMAND} -E remove_directory "<SOURCE_DIR>/submodules"
         COMMAND ${CMAKE_COMMAND} -E remove_directory "<SOURCE_DIR>/build_system"
-        COMMAND sh -c
-                "${GIT_EXECUTABLE} apply --verbose ${CMAKE_CURRENT_LIST_DIR}/${SEQAN3_LAST_STABLE_VERSION}/*.patch"
+        COMMAND sh -c "${git_apply}"
         TEST_BEFORE_INSTALL TRUE
         TEST_COMMAND ${CMAKE_CTEST_COMMAND} --output-on-failure -j2
         INSTALL_COMMAND "")
 endmacro ()
 
 if (EXISTS "${CMAKE_CURRENT_LIST_DIR}/${SEQAN3_LAST_STABLE_VERSION}")
-    seqan3_api_stability_test ("snippet")
-    seqan3_api_stability_test ("unit")
+    set (SEQAN3_GIT_APPLY "${GIT_EXECUTABLE} apply --verbose ${CMAKE_CURRENT_LIST_DIR}/${SEQAN3_LAST_STABLE_VERSION}/*.patch")
 else ()
-    message (FATAL_ERROR "No patches found in ${CMAKE_CURRENT_LIST_DIR}/${SEQAN3_LAST_STABLE_VERSION}:")
+    set (SEQAN3_GIT_APPLY "true")
```

Comment:
What is this used for?
If I follow the code to line 34, this would lead to a `sh -c true` call...sure we can do that! Accepted!

Reply:
Yeah, git apply will fail if there are no patches. I want git apply to fail when there are patches but they cannot be applied. And I want the test to be built even when there are no patches.